lib/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fs = require('fs');
+const rewriter = require('./rewriter');
+
+/**
+ * @param {string} filePath - the path to the file
+ */
+function rewriteFile(filePath) {
+  const file = {
+    path: filePath,
+    source: fs.readFileSync(inputPath, 'utf8'),
+  };
+  rewriter(file);
+}
+
+// rewriter execution
+const inputPath = process.argv[2];
+console.log(inputPath);
```

Comment:
We probably don't want to commit this, or if we do, it'd be something more like `Rewriting ${inputPath}` inside `rewriteFile`.

I think on balance it's better to not print anything when your program succeeds. I will try to find a source for that "rule" somewhere tomorrow...

Reply:
I don't know a source for it but I agree. If every tool announced success then scripts that combine tools would be full of success output and important warnings in the output would be hard to find.